src/data/split_spans_csvs.py

```diff
@@ -1,27 +1,30 @@
-"""Split large spans.csv files in the concepts directory into separate files, to make them commitable to git."""
+"""Split large spans.csv files in the concepts directory into separate files, to make them commitable to git. First argument is optional filename to split, which defaults to "spans.csv" if not specified."""
 
 from pathlib import Path
 import logging
+import sys
 
 import pandas as pd
 
 LOGGER = logging.getLogger(__name__)
 LOGGER.setLevel(logging.INFO)
 
-MAX_SPANS_PER_FILE = 50000
+MAX_SPANS_PER_FILE = 50_000
```

Comment:
Is this a typo?

Reply:
no, i discovered [PEP 515](https://docs.python.org/3/whatsnew/3.6.html#whatsnew36-pep515) 🥳